### ISSUE TITLE
Ly doesn't clear tty after logging out

### ISSUE THREAD

**Issue by NGSpace** [fairyglade/ly#696]
### Pre-requisites

- [X] I have looked for any other duplicate issues

### Ly version

1.0.2-1

### Observed behavior

![20240927_010930](https://github.com/user-attachments/assets/1bccd552-bedc-4c23-bd08-f2c5e3e74d97)

(Don't mind my main pc in the background!)

### Expected behavior

Ly should've cleared the tty before drawing the login box

### Steps to reproduce

1. Log in to hyprland
2. Log out of hyprland (using wlogout)
3. Observe the beauty of the above image

### Relevant logs

```shell
idk how to check for logs of ly...
```

**Comment by shogeki** [comment]
I ended up editing /usr/share/wayland-sessions/hyprland.desktop as a workaround for this

``` Exec=Hyprland &> /dev/null
```

**Comment by NGSpace** [comment]
Doesn't that just delete all the logs? seems kinda dangerous.

**Comment by shogeki** [comment]
No, this only affects terminal output and has nothing to do with logging.
